libtiledbvcf/src/c_api/tiledbvcf.cc

```diff
@@ -870,6 +893,13 @@ int32_t tiledb_vcf_writer_get_last_error(
   return TILEDB_VCF_OK;
 }
 
+int32_t tiledb_vcf_writer_set_memory_budget(
+    tiledb_vcf_writer_t* writer, uint64_t size) {
+  writer->writer_->set_memory_budget(size);
```

Comment:
Do you need the sanity checks and save_error_catch?

Reply:
Right. Shall I add it to `tiledb_vcf_writer_set_scratch_space` right below?

Reply:
Yes good catch on that one missing it too.